updater/src/updater.rs

```diff
@@ -600,13 +611,15 @@ impl<O: OperationsClient, F: HashFetch, T: TimeProvider, R: GenRange> Updater<O,
 
 		// We rely on a secure state. Bail if we're unsure about it.
 		if self.client.upgrade().map_or(true, |c| !c.chain_info().security_level().is_full()) {
-			return;
+			info!("The `security_level` check is DISABLED!!!!! It is just diabled to test the updater-verification and should be removed later, {}:{}", file!(), line!());
```

Comment:
Should we have a `const DEBUG: bool = ` for testing? Then such code could be committed and would help others working on this in the future.